src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,35 +4,100 @@
 
 @import '../includes/lib';
 
+$field-focus-ring: 0 0 0 2px rgba($color-blue-40, .5);
+$focus-border-color: #008aea;
```

Comment:
Should this be a token from the palette?

Reply:
I couldn't find a matching colour in the pallet.